### PR TITLE
Inverse styles for disabled checkboxen

### PR DESCRIPTION
**Pull request by daiyiwhale**
Previously:

Now:

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @jfurrow

File: src/Form/form.less [hunk 1]
```diff
@@ -222,6 +222,15 @@ base/forms.less
         input[type="checkbox"]  + .form-element-checkbox-decoy {
           box-shadow: inset 0 0 0 1px @form-control-default-inverse-border-color;
         }
+
+        input[type="checkbox"] {
+
+          &[disabled] + .form-element-checkbox-decoy {
```
Comment (jfurrow) [inline src/Form/form.less]: #nitpick Can we use `&:disabled` instead of the attribute selector?

**Comment by jfurrow** [comment]
This looks great!

File: src/Form/form.less (hunk 1 above)
Comment (daiyiwhale) [inline src/Form/form.less]: done!

**Comment by daiyiwhale** [comment]
:D

**Comment by rcorral** [comment]
Awesome, thanks for doing this @daiyiwhale  :)

**Comment by daiyiwhale** [comment]
:sparkles:

**Pull request merged**